### PR TITLE
CB-120: Remove unknown result in saved cohort search

### PR DESCRIPTION
**Pull request by jomadoye**
# JIRA TICKET NAME:
CB-120: [Remove unknown result in saved cohort search](https://issues.openmrs.org/browse/CB-120)

## SUMMARY:
This PR fixes unknown search result in the saved cohort search.

### CODE REVIEW

File: app/js/components/common/savedResultsTable.js [hunk 1]
```diff
@@ -28,11 +29,13 @@ const SavedResultsTable = ({
           </thead>
           <tbody>
             {
-              results.map((result, index) => {
-                const { uuid, description } = result;
-                return (
+              results.map((result) => {
+                if (result.name != 'appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments') {
```
Comment (paulupendo) [inline app/js/components/common/savedResultsTable.js:33]: Is there a reason why this result exists in the dataset?

Comment (wanjikum) [reply app/js/components/common/savedResultsTable.js:33]: Is there another way of solving it apart from hiding it?

Comment (jomadoye) [reply app/js/components/common/savedResultsTable.js:33]: @paulupendo @wanjikum This result comes from the database, and also it cannot be deleted from the database via the frontend. I dont know exactly what it is but I know it is not a saved cohort.

Comment (jomadoye) [reply app/js/components/common/savedResultsTable.js:33]: @dkayiwa please do you know what this (`appointmentschedulingui.appointmentDataSetDefinition.dailyAppointments`) is?

Comment (dkayiwa) [reply app/js/components/common/savedResultsTable.js:33]: Do you mean that a not saved cohort shows up in the list of saved cohorts?

Comment (dkayiwa) [reply app/js/components/common/savedResultsTable.js:33]: By the way, which exact rest url returns this? Are you able to reproduce it from here? https://modules-refapp.openmrs.org/openmrs/module/webservices/rest/apiDocs.htm

**Pull request closed without merging**